app/src/main/java/kth/jjve/xfran/EditEventActivity.java

```diff
@@ -36,21 +42,23 @@ protected void onCreate(Bundle savedInstanceState) {
         eventStartTimeEdit = findViewById(R.id.eventStartTimeEdit);
         eventEndTimeEdit = findViewById(R.id.eventEndTimeEdit);
         Button buttonSave = findViewById(R.id.eventSave);
+        Button buttonClose = findViewById(R.id.close);
+        Button buttonExport = findViewById(R.id.savetogoogle);
 
         String s_date = "Date: " + CalendarUtils.cleanDate(CalendarUtils.selectedDate);
         eventDate.setText(s_date);
 
         /*-------- LISTENERS ------------*/
-        buttonSave.setOnClickListener(v -> {
-            saveEvent();
-        });
+        buttonClose.setOnClickListener(v -> { finish(); });
```

Comment:
I don't think that the stuff after the lambda actually needs to be in {}. I think it can just be the following
`buttonClose.setOnClickListener(v -> finish());`